src/payment/checkout/Checkout.test.jsx

```diff
@@ -11,11 +11,37 @@ import '../__factories__/basket.factory';
 import '../__factories__/userAccount.factory';
 import { transformResults } from '../data/service';
 
-jest.mock('@edx/frontend-analytics', () => ({
+jest.mock('@edx/frontend-platform/analytics', () => ({
   sendTrackEvent: jest.fn(),
 }));
 
-import { sendTrackEvent } from '@edx/frontend-analytics'; // eslint-disable-line import/first
+configureI18n({
+  config: {
+    ENVIRONMENT: process.env.ENVIRONMENT,
+    LANGUAGE_PREFERENCE_COOKIE_NAME: process.env.LANGUAGE_PREFERENCE_COOKIE_NAME,
+  },
+  loggingService: {
+    logError: jest.fn(),
+    logInfo: jest.fn(),
+  },
+  messages: {
+    uk: {},
+    th: {},
+    ru: {},
+    'pt-br': {},
+    pl: {},
+    'ko-kr': {},
+    id: {},
+    he: {},
+    ca: {},
+    'zh-cn': {},
+    fr: {},
+    'es-419': {},
+    ar: {}
+  },
+});
+
+import { sendTrackEvent } from '@edx/frontend-platform/analytics'; // eslint-disable-line import/first
```

Comment:
I think in a recent version of jest they added handling to hoist `jest.mock` calls to the top of the scope, which made it unnecessary to import things "after" they're `jest.mock`'d.  Point being, we could move this up and remove the eslint-disable-line comment if we wanted.

Reply:
updated.